packages/mobile/src/components/core/Text.tsx

```diff
@@ -0,0 +1,25 @@
+import { Text as TextBase, TextProps as TextBaseProps } from 'react-native'
+
+import { makeStyles, typography } from 'app/styles'
+
+type TextProps = TextBaseProps & {
+  variant: keyof typeof typography
+  noGutter?: boolean
+}
+
+const useStyles = makeStyles(
+  ({ typography, palette }, { variant, noGutter }) => ({
+    root: {
+      ...typography[variant],
+      color: palette.neutral,
+      ...(noGutter && { marginBottom: 0 })
+    }
+  })
+)
+
+export const Text = (props: TextProps) => {
```

Comment:
Should we add a `@deprecated` jsdoc comment to our other `Text` component?

Reply:
down at some point, didnt want to make too much of an assumption there, was hoping to try out this other text component for a sec to see how it goes? will add a note though.